web/modules/asu_modules/asu_brand/.gitignore

```diff
@@ -12,18 +12,18 @@ node_modules/@asu-design-system/*
 !node_modules/@asu-design-system/component-cookie-consent/
 node_modules/@asu-design-system/component-cookie-consent/*
 !node_modules/@asu-design-system/component-cookie-consent/dist
-!node_modules/@asu-design-system/component-header
-node_modules/@asu-design-system/component-header/*
-!node_modules/@asu-design-system/component-header/dist
+!node_modules/@asu-design-system/component-header-footer
+node_modules/@asu-design-system/component-header-footer/*
+!node_modules/@asu-design-system/component-header-footer/dist
 
```

Comment:
`@asu-design-system` was our old scope before we started using GitHub packages.  Lines 11-17 could just be removed at this time.